build/img_spriting.js

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Since we are not using sprites I think this can be removed, as well as the related parts in `gulpfile.js`